.github/pull_request_template.md

```diff
@@ -34,5 +29,42 @@ _Please delete options that are not relevant._
 - [ ] I have commented my code, particularly in hard-to-understand areas
 - [ ] I have created a helpful and easy to understand `README.md`
 - [ ] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
-- [ ] My changes generate no new warnings
-- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.
+- [ ] I have added the project meta data in the PR template.
+- [ ] I have created the ``requirements.txt`` file if needed.
+
+## Project Metadata
+
+`` If there is no-file/nothing to fill the below fields with, then type: none ``
+
+`` Example: `` If no requirements.txt needed/present then type in ``Requirments`` none => ``Requirments: none``
```

Comment:
Umm, I have a question here, According to the regex rule you have in line 10:
`requirments_path = r"Requirments: (.+)"`

I think it will capture this value irrespective of the case where I am providing a path to file in line 62 in this PR template. This will make every entry in the database none for requirements. I think you can restructure this line as:
```suggestion
`` Example: `` If no requirements.txt needed/present, then type none in ``Requirments``.
```

Reply:
Yes, I realized this after running through multiple test